ui/app/components/evaluation-sidebar/detail.js

```diff
@@ -0,0 +1,78 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import d3 from 'd3';
```

Comment:
If we want to be more specific about our imports here, we could do `import { stratify } from 'd3-hierarchy'` and use `stratify()` without the `d3.` prefix below. The utility of this is probably dependent on tree-shaking of D3 at import time.